fastchat/conversation.py

```diff
@@ -847,17 +847,17 @@ def get_conv_template(name: str) -> Conversation:
 register_conv_template(
     Conversation(
         name="llama-2",
-        system="<s>[INST] <<SYS>>\nYou are a helpful, respectful and honest assistant. Always answer as helpfully as possible, while being safe.  "
+        system="<s>[INST] <<SYS>>\nYou are a helpful, respectful and honest assistant. Always answer as helpfully as possible, while being safe. "
```

Comment:
there are two spaces right? according to
https://github.com/facebookresearch/llama/blob/cfc3fc8c1968d390eb830e65c63865e980873a06/llama/generation.py#L47

Reply:
or that's a typo in llama repo?